src/Services/NewsletterService.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Services;
+
+use Blumilk\Meetup\Core\Models\NewsletterSubscriber;
+use Illuminate\Http\Response;
+
+class NewsletterService
+{
+    public function isSubscriber(NewsletterSubscriber $subscriber): Response
+    {
+        if ($subscriber->getAttribute("status") === 1) {
+            return response("You are already subscriber. Would you like to change your preferences?");
+        }
+        $subscriber->status = 1;
```

Comment:
Why are you using a number here in the status field, and below is a boolean value (`unsubscribe` method - `false`)